### PR TITLE
HACKING.md: clarify meaning and provide proper usage of OUTPUT_COVERAGE variable

### PR DESCRIPTION
**Pull request by php-coder**
@mfojtik PTAL.

Fix for #5797

### CODE REVIEW

**Comment by mfojtik** [comment]
LGTM @bparees merge?

File: HACKING.md [hunk 1]
```diff
@@ -100,7 +100,9 @@ To run tests without the go race detector, which is on by default, use:
 A line coverage report is run by default when testing a single package.
 To create a coverage report for all packages:
 
-    $ OUTPUT_COVERAGE=true hack/test-go.sh pkg/build
+    $ OUTPUT_COVERAGE=/path/to/dir hack/test-go.sh pkg/build
+
+After that you could open `/path/to/dir/coverage.html` in the browser.
```
Comment (kargakis) [inline HACKING.md]: s/could/can

Comment (php-coder) [inline HACKING.md]: @kargakis fixed, thanks!

**Comment by bparees** [comment]
@mfojtik lgtm but nothing is merging into origin until 3.1 is done.

**Comment by mfojtik** [comment]
@bparees that's why I was asking you :-)

**Comment by php-coder** [comment]
Could we merge it or not yet?

**Comment by bparees** [comment]
yes we can.
[merge]

**Comment by php-coder** [comment]
[test]

**Comment by php-coder** [comment]
@mfojtik @bparees Now it's green! Could try to you merge it again?

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**